common/src/main/java/com/wynntils/utils/wynn/InventoryUtils.java

```diff
@@ -60,6 +60,22 @@ public static List<ItemStack> getAccessories(Player player) {
         return accessories;
     }
 
+    public static boolean isItemListsEqual(List<ItemStack> firstItems, List<ItemStack> secondItems) {
+        if (firstItems.size() != secondItems.size()) return false;
+
+        for (int i = 0; i < firstItems.size(); i++) {
+            var newItem = firstItems.get(i);
+            var oldItem = secondItems.get(i);
+            if (!newItem.getItem().equals(oldItem.getItem())
+                    || newItem.getDamageValue() != oldItem.getDamageValue()
+                    || newItem.getCount() != oldItem.getCount()
+                    || !ItemStack.tagMatches(oldItem, newItem)) {
```

Comment:
Doesn't ItemStack#equals do this which means firstItems.equals(secondItems) does this?

Reply:
ItemStack doesn't implement equals... Okay.

Reply:
Yep, I noticed that. Hence this method. I hope that comparing item type, damage, count and lore that I capture everything that is relevant.